lib/tapioca/dsl/compilers/rspec.rb

```diff
@@ -54,13 +71,400 @@ def create_includes(klass)
           end
         end
 
+        class SemanticNode < T::InexactStruct
```

Comment:
What does T::InexactStruct do? It seems to be an undocumented, private Sorbet class. Is using this really necessary?